.gitignore

```diff
@@ -8,3 +8,4 @@ docs/src
 *.tmp
 *.swo
 *.swp
+.vscode
```

Comment:
Text editor specific ignores should be in your own global `.gitignore` :)

Reply:
lol I never thought of this, you just made my day :)